public/js/util/validators.js

```diff
@@ -15,3 +15,13 @@ export const isHttpsUrl = (value) => {
     return Promise.resolve(error);
   }
 };
+
+export const isValidTimelineDate = (date) => {
+  const stringValue = typeof date === 'string' ? date : '';
+    if (stringValue.match(/^\d{2}[./-]\d{2}[./-]\d{4}$/)) {
+        return Promise.resolve(true);
+    } else {
+      const error = new FieldError('invalid-date', 'Date not in a valid format of dd/mm/yyyy.');
```

Comment:
Let's output the value of `date` too, it might help debug problems in the future.